Automattic/wp-calypso
# Jetpack Offer Reset: Show Real-time and Daily options in product description.

This PR is continuation from [Jetpack Offer Reset: Show product options in product description (daily or real-time).](https://github.com/Automattic/wp-calypso/pull/44925)
#### Changes proposed in this Pull Request

This PR adds the text, "Available options: Real-Time and Daily" to the product description, IF:
- The product offers a real-time and daily option.
- and if the product is not already owned by the user.
- and if the product is not bundled in another plan.

#### Testing instructions

- Ensure your sandbox site does't have any Jetpack plans/products purchased.
- Enable the _Offer Reset Flow_ in _A/B tests_.
- Visit the Plans section.

Look at the description section in each of the Plan/Individual Product cards.  Every product/plan that offers a "real-time" and "daily" option will have, "Available options: Real-Time or Daily." appended to the description.  In this case, "Jetpack Backup" and "Jetpack Security" with have the appended text because those are the only plans that offer real-time and daily options at this point.

Screenshots:
![Markup 2020-08-25 at 12 36 01](https://user-images.githubusercontent.com/11078128/91219529-9fb0bc00-e6cf-11ea-8e8c-f90a194b1ed1.png)

Next, purchase Jetpack Security, reload the plans page and you'll no longer see the the appended text ("Available options: Real-Time or Daily.") in both Jetpack Security and Jetpack Backup (because Backup is included with Security).
![Screenshot on 2020-08-25 at 12-32-40](https://user-images.githubusercontent.com/11078128/91219607-bfe07b00-e6cf-11ea-9704-e22af20d3027.png)

Next, remove Jetpack Security, and then purchase Jetpack Backup. Then reload the Plans page. You should see the appended text ("Available options: Real-Time or Daily.") in Jetpack Security, but no longer see the appended text ("Available options: Real-Time or Daily.") in the product description of Jetpack Backup (because you own it). 

![Screenshot on 2020-08-25 at 12-34-09](https://user-images.githubusercontent.com/11078128/91219658-d555a500-e6cf-11ea-815a-c20033ca89bd.png)
